minions/async/src/main/resources/application.properties

```diff
@@ -46,6 +49,6 @@ amqp.username=microcks
 amqp.password=microcks
 
 # Configure the minion own behavioral properties.
-minion.supported-bindings=KAFKA,WS
+minion.supported-bindings=KAFKA,WS,NATS
```

Comment:
Can you please remove NATS from the defaults so that we keep a small default setup?